### ISSUE TITLE
tab / enter navigation on bulk upload labeling

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#3791]
A followup to PR #3789 

Revision request: For a team bulk upload, can we use 'enter' (or 'tab') to move to the next box (the next team member).  That is don't submit.  Except...  on the last box of the team, let's make enter submit for the team.

So, if I've specified the max numbers per team to be 4, there will be 4 boxes.  I type 'al', use downarrow to select "alyssa p hacker", press enter, advancing to the second box, type 'bit', downarrow to select "ben bitdiddle", press enter, advancing to the third box, if that's the whole team I press enter, going to the 4th box, and then (since I'm on the final box for that team) enter again will submit.

VPAT:  The page is mostly ok for keyboard navigation, but sometimes we lose focus on the button.  If it's a resubmit for a student, the popup for rename/overwrite comes up, but I can't press 'enter' to submit.  In fact, tab and enter navigate us on the page under the popup.  There might be other issues as well.  Once you think it's good, let's get @pbailie to look over the finished UI.

We can talk more at the meeting tomorrow if this is unclear or not quite possible

**Comment by pbailie** [comment]
From the description, these seem to be the views being discussed (screenshots are from my WAVE report).
![Screen Shot 2019-06-07 at 8 07 11 PM](https://user-images.githubusercontent.com/12703624/59139232-d336e780-895f-11e9-9baa-01dd664b9067.png)
![Screen Shot 2019-06-07 at 8 07 21 PM](https://user-images.githubusercontent.com/12703624/59139234-d5994180-895f-11e9-9b36-a1e61165b519.png)

Errors:

1. The WAVE report shows that there are multiple label errors (the red icons in both screenshots).  This is primarily caused by missing `<label>` tags.  Those controls have no context in a screen reader.  e.g. "Split by Q.R. Code" is only spoken as "Checkbox.  Unchecked".  It does not speak the label "Split by Q.R. Code", so it appears as an undefined extraneous checkbox to a screen reader user.  

    The "User ID" text fields in the second screenshot can alternatively be described with an `aria-label` property (instead of `<label>`), so the visual UI does not need to change.

2. Another problem is the `aria-hidden="true"` on the PDF popup button.  That makes the control invisible to a screen reader.  The unfortunate bit is that there are numerous code examples around the Internet that show using `aria-hidden="true"` on font-awesome icons, but this is actually a bad practice.  Also, I am unable to access the control by keyboard, even with the screen reader off.

Warnings (yellow icons):

1. The radio buttons, top of the first screenshot, are not enclosed in `<fieldset>` / `<legend>`.

2. The `<h3>` headers are not following proper header heiarchy.  I.e. there is no `<h2>` anywhere.

---

As for the new functionality (pressing enter at the end of the student list to auto-submit), I feel that it may potentially confuse screen reader users as a UX (user experience).  But fundamentally, the UI seems OK.  Tab works just fine to get to the submit and delete buttons, and my screen reader speaks the buttons.

I am far more concerned by the points mentioned above, but I'd still like to discuss this UX with @mcglow2 (our next meeting is Monday June 10).
